dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/MetadataBitstreamRestRepository.java

```diff
@@ -243,10 +308,10 @@ private List<FileInfo> processInputStreamToFilePreview(Context context, Bitstrea
             fileInfos.add(new FileInfo(data, false));
         } else {
             String data = "";
-            if (bitstream.getFormat(context).getExtensions().contains("zip")) {
+            if (bitstream.getFormat(context).getMIMEType().equals("application/zip")) {
```

Comment:
I remember, that we should check it following the `getExtensions` because of some reason I do not remember. Please can we check it like that again?

Reply:
I don't understand your comment....